ppl/lake/lake.ts

```diff
@@ -62,19 +50,14 @@ function zedCommand(): string {
 
 export class Lake {
   lake: ChildProcess
-  root: string
 
-  constructor(rootDir: string) {
-    this.root = rootDir
-  }
+  constructor(public root: string, public port: number, public logs: string) {}
```

Comment:
This is an improvement. The local lake port and logs are now configurable. This is what enables us to run the system tests in parallel.